docs/scripting.md

```diff
@@ -678,7 +678,7 @@ module.exports = (robot) ->
         # Command is being executed too quickly!
         done()
       else
-        next () ->
+        next ->
```

Comment:
Just cleaning up a random style thing I happened to notice. Completely unrelated.